lib/train/extras/command_wrapper.rb

```diff
@@ -162,7 +162,13 @@ class CommandWrapper
     include_options WindowsCommand
 
     def self.load(transport, options)
-      if transport.os.unix?
+      if transport.class == Train::Transports::SSH::Connection && !transport.connected?
+        # Need to call run_command here to get the message and show it to user while raising exception
+        msg = transport.run_command("whoami").stdout
```

Comment:
Presuming that we _are_ in an error state, the use of `whoami` is arbitrary (we might be trying to connect to windows or other systems). It might be clearer to use a command like "does_not_exist".